next.config.js

```diff
@@ -16,6 +16,10 @@ const nextConfig = {
     })
     return config
   },
+  i18n: {
+    locales: ['en', 'es', 'nl', 'hi'],
+    defaultLocale: 'en',
+  },
```

Comment:
Strange, this seems to be breaking in the vercel deployment but it doesn't impact me when this change is made locally

Reply:
I just experimented it a bit more and It seems like this change is breaking a lot of other things as well.
such as challenges are not working: https://saving-satoshi-git-fork-vinayaksh42-issue312-savingsatoshi.vercel.app/en/chapters/chapter-2/mining-1?dev=true
Even on localhost the challenges are not working properly. Will look into this issue, until that I am changing this PR to draft.

Reply:
Does the metadata addition only function if these configs are added as well?